RandDenner/Cataclysm-DDA
# item_contents tests now use TEST_DATA instead of main game data

#### Summary
Infrastructure "Run item_contents tests with TEST_DATA items"


#### Purpose of change
Preserve assumptions in item_contents_test to avoid breaking builds.


#### Describe the solution
All items used in the 3 tests in this file, now have copies in the TEST_DATA directory. Now when the main items are edited, it shouldn't impact the test.


#### Describe alternatives you've considered



#### Testing
1. Change crowbar to small weight (200)
2. Run the item_contents test
3. Observe test pass


#### Additional context
